rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -174,6 +174,7 @@ BACKUP_SUFFIX="`date +%Y%m%d%H%M%S`.ldif"
 # Other
 TIMEOUT="60"    # Max time to stop process
 FD_LIMIT="2048" # Max file descriptor
+VM_LIMIT="108490720" # Max virtual memory
```

Comment:
I will move this setting to rudder-slapd.conf, allowing override from the default file